### ISSUE TITLE
Doesn't work with Python3

### ISSUE THREAD

**Issue by isuckatprogram** [minimaxir/gpt-2-simple#241]
I'm trying to run the example on my machine but I'm having some weird issues. I used a tool which automatically installs all the needed python packages but its still not working.

The python console:

```python
ModuleNotFoundError: No module named 'tensorflow.contrib'
```

**Comment by rbewoor** [comment]
This is because you are trying to implement coffee within for TF1.X in a TF2.X environment. You need to use the exact version minimaxir specified. Contrib not available in TF2.

**Comment by WAUthethird** [comment]
Use Python 3.6.

**Comment by minimaxir** [comment]
Yes it has to be TF1.X. (I thought there was an assert to that effect? weird)
